apps/namadillo/src/App/Transfer/routes.ts

```diff
@@ -24,4 +27,5 @@ export default {
   shield,
   shieldAll,
   example,
+  exampleFromNamada,
```

Comment:
Sorry for the conflict here, but I believe you can render the component on these pages

`<ExampleFromNamada />`
- App/Transfer/Transfer.tsx
- http://localhost:5173/transfer

`<Example />`
- App/Ibc/IbcTransfer.tsx
- http://localhost:5173/ibc